components/support/error/src/macros.rs

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/// Tell the application to report an error
+///
+/// This will cause the error to be reporting to a Sentry-like system.  We will send extra context
```

Comment:
Really nitpicking here, but the comment implies this code will do the reporting. Maybe something like "If configured by the consuming application, this will be reported to that application, which may submit etc"?